free-2.md

```diff
@@ -11,41 +11,85 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+_Right now, Mudd freshman must use a Python interface to learn the made up,
+similar to assembly, programming language HMMM. This interface requires
+students to carefully construct their program because they cannot easily
+go back and change line numbers (that they must hand type for each line).
+Hopefully, my language would be able to make using HMMM a lot more 
+accessible and user-friendly._
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+_Right now HMMM is a language and students will hopefully still work with
+it as a language because that is what users are comfortable with. Hopefully,
+this will just be an improvement on the original language._
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
-
+_Addressing one of my biggest pet peeves of CS5/42 makes me really excited
+and knowing that it could help students feel more comfortable with a topic
+of the into CS class that can be confusing._  
+_I came up with it by talking to some friends. I asked for their help with
+something that is impossible in my language for free-1. One of them jokingly
+mentioned HMMM (which would have, indeed, been impossible). However, we 
+started talking about how terrible it was to program in HMMM because of the
+line numbers, the difficult debugging, and other problems. So, I figured
+that one option would be to rewrite HMMM, fixing these problems._
```

Comment:
Do you need a DSL, or better tooling? Maybe providing an IDE that handled a lot of that would solve the problem?